core/string/translation.h

```diff
@@ -32,6 +32,8 @@
 #define TRANSLATION_H
 
 #include "core/io/resource.h"
+#include "core/object/gdvirtual.gen.inc"
+#include "core/object/script_language.h"
```

Comment:
Why is this include needed? The macros used in this PR seem to come from `core/object/object.h` and `core/object/gdvirtual.gen.inc`.

Reply:
I searched for incomplete class `ScriptInstance` and found that include.

Reply:
Changed to `core/object/object.h`

Reply:
Ah I see, the include for `ScriptInstance` should likely be added to `core/object/gdvirtual.gen.inc` itself.

`core/object/object.h` is probably not needed here as it's already included by `core/io/resource.h` (though it's not wrong to have it here either).

Reply:
> Ah I see, the include for ScriptInstance should likely be added to core/object/gdvirtual.gen.inc itself.

That's outside the scope of this PR though, I can have a look at this myself. Feel free to put back `script_language.h` here since it's needed.